README.md

```diff
@@ -7,7 +7,7 @@ Package license: MIT
 
 Feedstock license: BSD 3-Clause
 
-Summary: Definitions for the "X fixes" protocol.
+Summary: "X fixes" extension C prototypes.
```

Comment:
Update based on the [`summary` change]( https://github.com/conda-forge/xorg-fixesproto-feedstock/commit/667fc965dbabefd87c9e86c7be6969d17b566773#diff-e178b687b10a71a3348107ae3154e44cR43 ) in the recipe.